src/loaders/assetLoader.js

```diff
@@ -36,14 +36,15 @@ module.exports = async function assetLoader() {
     // Asset is not an image
   }
 
+  const pathSepPattern = new RegExp(`\\${path.sep}`, 'g');
```

Comment:
It would, but I haven't tested it yet.

If `path.sep === '/'`, the generated RegExp would be `/\//g`.